include/alpaka/mem/view/ViewPlainPtr.hpp

```diff
@@ -90,7 +89,8 @@ namespace alpaka
                 //-----------------------------------------------------------------------------
                 auto operator=(ViewPlainPtr &&) -> ViewPlainPtr & = default;
                 //-----------------------------------------------------------------------------
-                ~ViewPlainPtr() = default;
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC ~ViewPlainPtr() = default;
```

Comment:
This is somehow inconsistent now. removed on the constructor, adde on the destructor.

Reply:
weird, it should be on the ctor as well, why did I remove it? :/